app/routes/questionnaire.py

```diff
@@ -315,19 +316,18 @@ def get_thank_you(schema, session_store):
 @with_schema
 @with_session_store
 def send_confirmation_email(session_store, schema):
-    if not session_store.session_data.confirmation_email_count:
```

Comment:
This has been removed as the new design means you can go Thank you -> confirm (answer no) -> send_confirmation_email. This means there would be no count and therefore it now can't return a 404